editor/locale/lang.vi.js

```diff
@@ -79,7 +79,7 @@ svgEditor.readLang({
 		"fit_to_sel": "Phù hợp để lựa chọn",
 		"align_relative_to": "Căn liên quan đến ...",
 		"relativeTo": "liên quan đến:",
-		"Sửa": "Sửa",
+		"page": "Page",
```

Comment:
Ahh, I didn't even check the translation of this one. I thought it was just a mistake. Ok, this seems better than was there before, as we shouldn't be milking any pages in svgedit (here, and the other "no idea" comments on this review). Lets merge this :)